python/hail/tests/tests.py

```diff
@@ -379,27 +379,25 @@ def test_dataset(self):
                        .key_by("locus"))
 
         skatVds = (vds2.split_multi()
-            .annotate_variants_table(intervalsSkat, root="va.genes", product=True)
+            .annotate_variants_table(intervalsSkat, root="va.gene")
             .annotate_variants_table(weightsSkat, root="va.weight")
             .annotate_samples_table(phenotypesSkat, root="sa.pheno")
             .annotate_samples_table(covariatesSkat, root="sa.cov")
             .annotate_samples_expr("sa.pheno = if (sa.pheno == 1.0) false else " +
                                    "if (sa.pheno == 2.0) true else NA: Boolean"))
 
-        (skatVds.skat(variant_keys='va.genes',
-                      single_key=False,
+        (skatVds.skat(key_expr='va.gene',
```

Comment:
To keep the test the same, shouldn't you keep `product=True` in the annotate_variants_table above and explode here?